src/shared/api/index.ts

```diff
@@ -1,19 +1,30 @@
+import type {
+  Account,
+  AssetDefinition,
+  Asset,
+  AssetSearchParams,
+  Block,
+  Domain,
+  Paginated,
+  PaginationParams,
+  TransactionSearchParams,
+  Transaction,
+  PeerStatus,
+} from '@/shared/api/dto';
+
 interface HTTPService {
```

Comment:
Actually, this whole interface doesn't bring benefit anymore. It was needed to ensure consistency between an actual API and a fake one.

So, it would make sense to re-export the `http.ts` module directly.